projects/ad4130_iio/README.rst

```diff
@@ -0,0 +1,374 @@
+AD4130 IIO Application
+====================
+
+Introduction
+-----------------
+
+This guide gives an overview of using the ARM platform(s) supported (default is Mbed) 
+firmware with Analog Devices AD4130 Evaluation board and SDP-K1 (or other compatible)
```

Comment:
should the closing parenthesis be after 'board' in the next line?

Reply:
I think they are correct. What I meant was:
... AD4130 Evaluation board and SDP-K1 (or other compatible) MCU controller board

 (or other compatible) - This indicates other boards such as STM32-Disco, etc